xml/vt_docker_quick.xml

```diff
@@ -663,4 +663,163 @@ RUN zypper -n in vim</screen>
+          allosws to create organizations made by different users.
+        </para>
+        <para>
+          This component is closed source.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          The authentication component: this is used to protect the images
+          stored inside of the Docker Hub. This validates all the push, pull and
+          search requests.
+        </para>
+        <para>
+          This component is closed source.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          The storage backend: this is where the actual Docker images are sent
+          and downloaded from. This is provided by the Docker registry.
+        </para>
+        <para>
+          As said before, this component is open-source.
+        </para>
+      </listitem>
+    </itemizedlist>
+   </sect2>
+
+   <sect2 xml:id="sec.docker.registry.installation">
+     <title>Installing and Setting Up Docker registry</title>
+
+     <procedure>
+      <step>
+       <para>
+        Install the <systemitem>docker-distribution-registry</systemitem> package:
+       </para>
+       <screen>sudo zypper install docker-distribution-registry</screen>
+      </step>
+      <step>
+       <para>
+        To automatically start the Docker registry at boot time:
+       </para>
+       <screen>sudo systemctl enable registry</screen>
+      </step>
+      <step>
+       <para>
+        Start the Docker registry:
+       </para>
+       <screen>sudo systemctl start registry</screen>
+      </step>
+     </procedure>
+
+     <para>
+       The Docker registry configuration is defined inside of
+       <systemitem>/etc/registry/config.yml</systemitem>.
+     </para>
+     <para>
+       With the default configuration the registry listens on ports 5000 and
+       stores the Docker images under
+       <systemitem>/var/lib/docker-registry</systemitem>.
+     </para>
+
+     <!--
+      fcastelli 2015-07-08: TODO: update once we create a TID illustrating the secure setup of
+      Docker registry and Portus.
+     -->
+     <para>
+       For more details about Docker registry and its configuration please
+       consult the official documentation at:
+       <link xlink:href="https://docs.docker.com/registry/"/>.
+     </para>
+  </sect2>
+
+   <sect2 xml:id="sec.docker.registry.limitations">
+     <title>Limitations</title>
+
+     <para>
+       The Docker registry has two major limitations. First of all it lacks
+       any for of authentication. That means everybody with access to the
+       Docker registry can push and pull images to it. That also includes the
+       possibility to even overwrite already existing images.
+     </para>
+
+     <para>
+       There is no way to see which images have been pushed to the Docker
+       registry. You can to manually take note of what is being stored inside
+       of it. There's also no search functionality. This makes collaboration
```

Comment:
I would re-phrase it like: "There is also no search functionality, which makes collaboration harder."